revocation/ocsp/ocsp.go

```diff
@@ -66,7 +67,7 @@ func CheckStatus(opts Options) ([]*result.CertRevocationResult, error) {
 		// Assume cert chain is accurate and next cert in chain is the issuer
 		go func(i int, cert *x509.Certificate) {
 			defer wg.Done()
-			certResults[i] = ocsp.CertCheckStatus(cert, opts.CertChain[i+1], certCheckStatusOptions)
+			certResults[i] = ocsp.CertCheckStatus(context.Background(), cert, opts.CertChain[i+1], certCheckStatusOptions)
```

Comment:
Can we create one `ctx` and share it in Mutiple goroutine?

Reply:
Updated.